nginx/static/batch.html

```diff
@@ -86,10 +86,11 @@ <h1>BFEX Search</h1>
 $('#table').bootstrapTable();
   function ajax(keyword){
   var approaches = {0:'generic',1:'rake',2:'textrank',3:'tf-idf'}
+  var url = "http://localhost:8901";
```

Comment:
So, this will still hit the local host, and won't work in 'production'. You need to find a way to have this point to localhost when running on dev, or to the server when running elsewhere. Sorry if I wasn't clear about that.